### PR TITLE
added contentful and settings

### PR DESCRIPTION
**Pull request by paddyduff31**

### CODE REVIEW

**Review by con-nihr — COMMENTED** [review]
I am concerned about the duplication of functionality between this project and the study-api.
Any thoughts about how best to share the common code?

**Comment by paddyduff31** [comment]
Yeah I had my concerns about that too.  Only place is really common but as a package can it hold services?

File: src/ScheduledJobs/JobHandlers/RtsSiteImportJobHandler.cs [hunk 1]
```diff
@@ -90,10 +96,11 @@ public async Task<bool> HandleAsync(RtsSiteImport source)
                     list = list.GroupBy(l => l.Pk).Select(l => l.First()).ToList();
 
                     await _repository.BatchInsertAsync(list);
-                    _logger.LogInformation($"Page {pageNumber} saved {list.Count} items to the DB");
+                    _logger.LogInformation("Page {PageNumber} saved {ListCount} items to the DB", pageNumber,
+                        list.Count);
 
                     pageNumber++;
-                    await Task.Delay(50);
+                    await Task.Delay(50, cancellationToken);
```
Comment (con-nihr) [inline src/ScheduledJobs/JobHandlers/RtsSiteImportJobHandler.cs:103]: As above

**Review by con-nihr — APPROVED** [review]

File: src/ScheduledJobs/JobHandlers/RtsOrganisationImportJobHandler.cs [hunk 2]
```diff
@@ -82,17 +89,18 @@ public async Task<bool> HandleAsync(RtsOrganisationImport source)
                     list.AddRange(results.Select(RtsDataMapper.MapTo));
                     list = list.GroupBy(l => l.Pk).Select(l => l.First()).ToList();
                     await _repository.BatchInsertAsync(list);
-                    _logger.LogInformation($"Page {pageNumber} saved {list.Count} items to the DB");
+                    _logger.LogInformation("Page {PageNumber} saved {ListCount} items to the DB", pageNumber,
+                        list.Count);
 
                     pageNumber++;
-                    await Task.Delay(50);
+                    await Task.Delay(50, cancellationToken);
```
[the remark this answers is not in the document]
Comment (paddyduff31) [reply src/ScheduledJobs/JobHandlers/RtsOrganisationImportJobHandler.cs]: I'm not sure without investigating, I don't think this is even used anywhere

**Comment by paddyduff31** [comment]
@con-nihr I only removed the Cancellation tokens at the top lambda levels and anything within the HandleAsync function and left them cascading down

**Comment by con-nihr** [comment]
> @con-nihr I only removed the Cancellation tokens at the top lambda levels and anything within the HandleAsync function and left them cascading down

I think too much has been pared back, you need something to cascade down.
For example, the harness actions will have access to a real `CancellationToken` from the request. The only thing that needs to change is the very top level lambda handler.

**Pull request merged**